src/control/ros/Conversions.cpp

```diff
@@ -281,11 +314,11 @@ std::unique_ptr<SplineTrajectory> toSplineJointTrajectory(
 
   // Map joint indices between jointTrajectory and space subspaces.
   std::map<size_t, size_t> rosJointToMetaSkeletonJoint;
+  std::set<size_t> specifiedMetaSkeletonJoints;
```

Comment:
Nitpick: `std::unorderd_set` would be better since we don't worry about the order here.